docker/scripts/cloud-init.cfg

```diff
@@ -37,12 +37,11 @@ write_files:
       Type=oneshot
       RemainAfterExit=true
       Environment="HOME=/home/mlcommonsservice"
-      ExecStartPre=/usr/bin/docker-credential-gcr configure-docker --registries us-central1-docker.pkg.dev
       ExecStartPre=mount --bind /var/lib/nvidia /var/lib/nvidia
-      ExecStartPre=mount -o remount,exec /var/lib/nvidia
-      ExecStartPre=/usr/bin/docker pull us-central1-docker.pkg.dev/training-algorithms-external/mlcommons-docker-repo/mlcommons:criteo
-
-      ExecStart=/usr/bin/docker run --rm --name=mlcommons --volume /var/lib/nvidia/lib64:/usr/local/nvidia/lib64 --volume /var/lib/nvidia/bin:/usr/local/nvidia/bin --device /dev/nvidia0:/dev/nvidia0  --device /dev/nvidia1:/dev/nvidia1 --device /dev/nvidia2:/dev/nvidia2 --device /dev/nvidia3:/dev/nvidia3 --device /dev/nvidia4:/dev/nvidia4 --device /dev/nvidia5:/dev/nvidia5 --device /dev/nvidia6:/dev/nvidia6 --device /dev/nvidia7:/dev/nvidia7 --device /dev/nvidia-uvm:/dev/nvidia-uvm --device /dev/nvidiactl:/dev/nvidiactl us-central1-docker.pkg.dev/training-algorithms-external/mlcommons-docker-repo/mlcommons:criteo
+      ExecStartPre=mount -o remountexec /var/lib/nvidia
```

Comment:
is this intentional?

Reply:
yes, the mount --bind thing isn't necessary and the rest of the commands have a different image name.